app/controllers/admin/users_controller.rb

```diff
@@ -2,7 +2,13 @@ class Admin::UsersController < Admin::AdminController
   before_action :set_selected_menu_items
 
   def index
-    @users = current_company.users.order(created_at: :desc).compact
+    if params[:role] && params[:role] == "admin"
```

Comment:
```suggestion
    if params[:role] && params[:role] == "admin"
```

Same as below, I think we can skip checking for `params[:role]`